### DISCUSSION TITLE
Feature Request: Add a directory to "python.analysis.extraPaths" via contextual menu

### DISCUSSION THREAD

**Discussion by erictraut** [microsoft/pylance-release#2587] [Enhancement]
Many users of Pylance struggle to configure their "python.analysis.extraPaths" setting. It would be useful to support a contextual menu in the workspace browser that allows users to add a directory to this setting. A similar feature is provided in PyCharm.

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
